agave_app/renderer.h

```diff
@@ -47,7 +98,8 @@ class Renderer
                  const LoadSpec& loadSpec,
                  // rendererMode ignored if renderer is non-null
                  renderlib::RendererType rendererMode = renderlib::RendererType_Pathtrace,
-                 QOpenGLContext* glContext = nullptr);
+                 QOpenGLContext* glContext = nullptr,
+                 const CaptureSettings* captureSettings = nullptr);
```

Comment:
the renderer can now be initialized with CaptureSettings, so that it can determine on its own whether a render is completed or not. (In particular, CaptureSettings holds on to a RenderDuration object)